packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS, LIST_ITEM_TYPES } from "../../constants/";
+
+const ICONS_KEYS = Object.keys(ICONS);
+
+const ListItem = ({
+  active,
+  children,
+  subtitle,
+  type,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
+  onClick,
+}) => (
+  <div
+    onClick={onClick}
+    className={cx("ListItem", {
+      "ListItem--subitem": type === LIST_ITEM_TYPES.SUB,
+      "ListItem--miniitem": type === LIST_ITEM_TYPES.MINI,
+      "is-active": active,
+      "is-clickable": !!onClick,
+      "ListItem-divider": !hideDivider,
+    })}
+  >
+    {leftIcon && (
+      <div className="ListItem-leftIcon">
+        <Icon icon={leftIcon} />
+      </div>
+    )}
+    <div>
+      <div
+        className={cx({
+          "ListItem-title": type !== LIST_ITEM_TYPES.MINI,
+          "ListItem-subtitle": type === LIST_ITEM_TYPES.MINI,
+        })}
+      >
+        {children}
+      </div>
+      {subtitle && <div className="ListItem-subtitle">{subtitle}</div>}
+    </div>
+    <div className="ListItem-valueContainer">
+      <span
+        className={cx({
+          "ListItem-title": type !== LIST_ITEM_TYPES.MINI,
+          "ListItem-subtitle": type === LIST_ITEM_TYPES.MINI,
+        })}
+      >
+        {value}
+      </span>
+      <span className="ListItem-itemIcon">
+        {(onClick || rightIcon) && (
+          <Icon icon={rightIcon || ICONS.IconArrowRight} />
```

Comment:
Shouldn't this be set as a `defaultProp`?

Reply:
Nice... but no :P

So `ArrowRight` is expected to be shown only when `onClick` is present.
If we put it in `defaultProps` it would be shown always, unless defined otherwise.
Not sure if I was clear enough?

Reply:
Yep, got it 👍